apps/s_server.c

```diff
@@ -2262,11 +2262,10 @@ static int sv_body(int s, int stype, int prot, unsigned char *context)
             BIO_ctrl(sbio, BIO_CTRL_DGRAM_MTU_DISCOVER, 0, NULL);
 
 # ifndef OPENSSL_NO_SCTP
```

Comment:
On another *side note* why not #ifdef IPPROTO_SCTP?

Reply:
`prot` will never be set to `IPPROTO_SCTP` if OPENSSL_NO_SCTP is defined

Reply:
> prot will never be set to IPPROTO_SCTP if OPENSSL_NO_SCTP is defined

Exactly!

Reply:
So, in my mind that makes it better to check `OPENSSL_NO_SCTP`.